app/models/comment.rb

```diff
@@ -2,7 +2,9 @@ class Comment < ApplicationRecord
     belongs_to :author, class_name: 'User', foreign_key: 'author_id'
     belongs_to :post
 
-    after_create :update_comment_counter
+    after_save :update_comment_counter
+
+    private
 
     def update_comment_counter
       post.update(comments_counter: post.comments.count)
```

Comment:
It would be nice if you make test for this method to make your project follow the given requirements 👍